cypress/e2e/managePatches.feature

```diff
@@ -0,0 +1,44 @@
+@common
+@property
+@managePatches
+
+@ManagePatches
+Feature: View and manage patch and area assignment
+	I want to view and manage patch and area assignment
+
+	Background:
+		Given I am logged in
+
+	Scenario: Navigate to manage patches from property, view details, return to property
+		Given I seeded the database with an asset
+		When I view the property in MMH
+		And the property's patch details are displayed
+		When I click on the manage patches button
+		Then the manage patches page is displayed
+		And the patches table is displayed
+		When I click on the back link
+		Then the property's patch details are displayed
+
+	Scenario Outline: Filter patches and areas list by area
+		When I visit the manage patches page directly
+		And I can see a row for patch '<patch>'
+		When I select '<area option>' from the area dropdown
+		Then I can see a row for patch '<patch>'
+		And I cannot see a row for patch '<notpatch>'
+		Examples:
+			| area option |  patch  |  notpatch  |
+			|     All     |   SN1   |            |
+			|   SN Area   |   SN1   |     SD1    |
+			|   SD Area   |   SD1   |     SN1    |
+	
+	# Skip until we release patch reassignment
+	# Scenario Outline: Switch patch assignments
```

Comment:
I wonder is we can use tags like `@ignore-this` to skip the test? 
https://stackoverflow.com/questions/58396278/how-to-skip-a-feature-in-cypress-so-that-its-not-tested-for-end-to-end-tests